imports/collections/schemas/core.js

```diff
@@ -0,0 +1,54 @@
+import SimpleSchema from "simpl-schema";
+
+export const Money = new SimpleSchema({
+  amount: {
+    type: Number,
+    min: 0
+  },
+  currencyCode: String
+});
+
+const SurchargeMessagesByLanguage = new SimpleSchema({
+  content: String,
+  language: String
+});
+
+/**
+ * @name Surcharge
+ * @memberof Schemas
+ * @type {SimpleSchema}
+ * @property {String} _id
+ * @property {Money} amount
+ * @property {String} cartId optional
+ * @property {String} fulfillmentGroupId optional
+ * @property {String} messagesByLanguage optional
+ * @property {String} reason optional
+ * @property {String} surchargeId optional
+ */
+export const Surcharge = new SimpleSchema({
```

Comment:
Should we call this `AppliedSurcharge` to match the GraphQL schema and to be clearer?